app/assets/templates/reshare_tpl.jst.hbs

```diff
@@ -6,7 +6,7 @@
 
       {{#with root}}
         <a href="/people/{{author.guid}}" class="img {{{hovercardable this}}}">
-          <img src="{{author.avatar.small}}" class="avatar" />
+          <img src="{{author.avatar.small}}" class="avatar micro" />
```

Comment:
Why did you add `micro` here? We set the height and width of those avatars to `30px` [here](https://github.com/diaspora/diaspora/blob/develop/app/assets/stylesheets/stream_element.scss#L129) so adding `micro` won't have any effect.